src/styles/constants/theme-print.scss

```
@@ -180,6 +180,11 @@
 					position: unset !important;
 					transform: unset !important;
 					text-align: initial;
+					break-inside: avoid;
+					page-break-inside: avoid; /*avoid widgets breaking on transition between pages*/
```

Stylelint (17.11.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)